packages/ipxe-bluebanquise/bluebanquise_dhcpretry.ipxe

```diff
@@ -33,6 +33,16 @@ echo
 echo (waiting 5s and chain...)
 sleep 5
 
-chain http://${next-server}/preboot_execution_environment/convergence.ipxe || shell
+echo Trying to chain... 1/3
```

Comment:
This change is not related to the update of the packaging process.

Reply:
I Know 🙁 
Simply missing time, so I did it here...